pkgs/tools/security/sbomnix/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, coreutils
+, curl
+, gnugrep
+, gnused
+, gzip
+, graphviz
+, nix
+, reuse
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "sbomnix";
+  version = "1.4.5";
+
+  src = fetchFromGitHub {
+    owner = "tiiuae";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-RxDFxVGivVBw2uhtzf231Q2HHTBFKSqGrknr2Es/ygM=";
+  };
+
+  makeWrapperArgs = [
+    "--prefix PATH : ${lib.makeBinPath [ coreutils curl gnugrep gnused gzip graphviz nix ]}"
+  ];
+
+  propagatedBuildInputs = [
+    reuse
+    python3Packages.colorlog
+    python3Packages.graphviz
+    python3Packages.numpy
+    python3Packages.packageurl-python
+    python3Packages.pandas
+    python3Packages.requests
+    python3Packages.tabulate
```

```suggestion
    colorlog
    graphviz
    numpy
    packageurl-python
    pandas
    requests
    tabulate
```

After the change suggested in all-packages

---

Fixed in 0f1b2e7747bb9dea653ca76d107028bc25450b22